data/hai/hai reveal 4 middle.txt

```diff
@@ -1519,26 +1879,53 @@ mission "Hai Reveal [A11] Sound the Alarm"
 			label next
 			`	Sayari gives you a meaningful look and whispers, "A lot of them paid attention to that..."`
 			`	The next Elder to their feet is an older lady Hai, and her expression alone is enough to cause a flicker of concern to disrupt Teeneep's careful mask of calm. She gestures around the room, pointing with a digit and squinting at several others in the room. "You are, all of you, cowards. This is the highest chamber of the Hai, elected to protect and serve, and to ensure the enduring stability of the great edifice which is now a hundred millennia of stability. This one here has sought to upend that authority and topple that edifice in favor of nothing more than a statistical blip. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone! Freedom through peace, peace through order, order through security, and security through secrecy. This is the mandate of this chamber, and no matter how virtuous the challenger may be, this threat cannot be allowed to stand!"`
+			action
+				"Hai Reveal [A11] Sound the Alarm: time" = 4
+			choice
+				`	(Remain silent and listen.)`
+					goto "remain silent and listen"
+				`	"That response was sickening. I'm going to the reading room."`
+				`	"If you'll pardon me, I have some reports to read."`
+			`	Before you can speak, Sayari puts her hand on your shoulder and shakes her head. "This is the moment, whatever happens next will decide what happens here," she whispers. "Be patient."`
+			label "remain silent and listen"
 			`	There is a growing rumble of assent as the crescendo of her impassioned statement crests to its conclusion. She levels a clawed finger directly at Teeneep and the room is filled with the angry cries of both those emboldened by her diatribe and those distressed by her implication. The moderator makes several attempts to restore order as over a dozen Elders jump to their feet, but the cacophony is not so easily restrained. At your side, Sayari's expression is ashen, but Teeneep seems to be seeking out a particular face somewhere in the room with a fierce intensity.`
```

Comment:
I think it makes more sense for choice to come after she points at Teeneep and is clearly finished.
```suggestion
			`	There is a growing rumble of assent as the crescendo of her impassioned statement crests to its conclusion. She levels a clawed finger directly at Teeneep and the room is filled with the angry cries of both those emboldened by her diatribe and those distressed by her implication. The moderator makes several attempts to restore order as over a dozen Elders jump to their feet, but the cacophony is not so easily restrained. At your side, Sayari's expression is ashen, but Teeneep seems to be seeking out a particular face somewhere in the room with a fierce intensity.`
			action
				"Hai Reveal [A11] Sound the Alarm: time" = 4
			choice
				`	(Remain silent and listen.)`
					goto "remain silent and listen"
				`	"That response was sickening. I'm going to the reading room."`
				`	"If you'll pardon me, I have some reports to read."`
			`	Before you can speak, Sayari puts her hand on your shoulder and shakes her head. "This is the moment, whatever happens next will decide what happens here," she whispers. "Be patient."`
			label "remain silent and listen"
```